ivy/functional/backends/tensorflow/layers.py

```diff
@@ -452,9 +452,7 @@ def conv_general_transpose(
             axis=-1,
         )
     elif dims == 2:
-        res = tf.concat(
-            [
-                conv2d_transpose(
+        res = conv2d_transpose(
```

Comment:
If the test passes this way, you can do the same for the 1D and 3D cases as well.